doc/nrf/protocols/matter/index.rst

```diff
@@ -24,6 +24,12 @@ The |NCS| allows you to develop applications with different versions of Matter,
 +--------------------------+-----------------------------------------------------+------------------------+
 | |release|                | :ref:`1.1.0 <ug_matter_overview_dev_model_support>` | 1.1.0.1                |
 +--------------------------+                                                     |                        |
+| v2.5.1                   |                                                     |                        |
```

Comment:
Shouldn't we list the newest 2.5.2 release here as well.?

Reply:
The `|release|` tag is replaced by the latest official release. It's doing some work there in case we forget to update for some releases.